src/pages/Privacy.tsx

```diff
@@ -0,0 +1,88 @@
+import { Component } from "solid-js";
+
+import "../style/legal.scss";
+
+const Privacy: Component = () => {
+    return (
+        <div class="privacy-container">
+            <h1>Privacy Policy</h1>
+
+            <h2>1. Introduction</h2>
+            <p>
+                This Privacy Policy applies to all services provided by Boltz
+                S.A. de C.V., an entity formed under the laws of El Salvador
+                (collectively "we," "our," "Boltz," or the "Service") and
+                describes how we collect, use, and disclose your information
+                when you interact with our services, including through our
+                website or any software interfacing with our API. If you
+                disagree with any terms herein, you must refrain from using our
+                Service.
+            </p>
+
+            <h2>2. Information We Collect</h2>
+            <p>
+                Below is a detailed overview of information collected during
+                your use of our services:
+            </p>
+            <p>
+                2.1 Swap Destination: We collect and store data related to
+                transaction destinations, such as Bitcoin addresses or Lightning
+                invoices.
+            </p>
+            <p>
+                2.2 Swap Origin: We collect and store data related to
+                transaction origins, including transaction identifiers or
+                Lightning invoices.
+            </p>
+            <p>
+                2.3 IP Address Handling: We do not permanently store your IP
+                address but may temporarily process it for taxation compliance
+                or geo-restriction enforcement. To use our services anonymously,
+                you may employ a VPN or use
```

Comment:
```suggestion
                you may employ a VPN or use{" "}
```
to fix the lack of space:
![image](https://github.com/user-attachments/assets/0e048eda-e397-45e1-bfb8-bdbcf1313d33)